### ISSUE TITLE
Request: Allow player to set their death link status if the game does not have a death link option

### ISSUE THREAD

**Issue by ScipioWright** [Eijebong/Archipelago-lobby#22]
As titled.

Some games have death link as an entirely client-side option (Jak & Daxter, Tunic), so there's no way to display the intent to use death link on the site.

Unless that exists already and I missed it then sorry about the invalid issue

**Comment by Eijebong** [comment]
A workaround for that would be to just put `death_link: true` in the YAML. The website would extract that as deathlink enabled and the world would ignore it.

A solution would be to mark those worlds as deathlink enabled all the time, but I definitely want to avoid adding more buttons for this. Happy to take ideas in if you have any.

**Comment by ScipioWright** [comment]
Gotcha, didn't realize that would work o:

Yeah, that seems like the only reasonable option here, I can't think of another one that isn't super finicky

**Comment by Eijebong** [comment]
Yeah deathlink is somewhat special, because there's no standard for it, I just "try my best" and on every YAML check for `death_link`, `deathlink` and `DeathLink`. It's the only attribute like that at the moment, see https://github.com/Eijebong/Archipelago-lobby/blob/32c9bcdba6e4379d891369397aa1b4ce21a89bc0/src/extractor/mod.rs#L224 and https://github.com/Eijebong/Archipelago-lobby/blob/main/src/extractor/mod.rs#L279

**Comment by ScipioWright** [comment]
Yeahhh, it's more of a "suggested" feature than anything. imo it belongs client-side, but unless your site connects to the room then you can't exactly check the tags so there's no reasonable way to get that working nicely
